### PR TITLE
server: add http api for get ddl history 

### PR DESCRIPTION
**Pull request by ciscoxll**
### What problem does this PR solve? 
From https://github.com/pingcap/tidb/pull/6245

Add `http api` query the all ddl history.

```shell
curl http://{TiDBIP}:10080/ddl/history
```

Get all ddl job history information can limit the number of displays.

```shell
curl http://{TiDBIP}:10080/ddl/history?limit={number}
```

### What is changed and how it works?

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tidb/8256)

### CODE REVIEW

**Comment by ciscoxll** [comment]
PTAL @winkyao @zimulala

File: docs/tidb_http_api.md [hunk 1]
```diff
@@ -159,4 +159,13 @@ timezone.*
     curl -X POST http://{TiDBIP}:10080/ddl/owner/resign
     ```
     
+1. Get TiDB all ddl job history information.
+	```shell
+	curl http://{TiDBIP}:10080/ddl/history
+	```
+
+1. Get TiDB all ddl job history information can limit the number of displays.
```
Comment (winkyao) [inline docs/tidb_http_api.md]: Get count {number} TiDB DDL job history information.

File: docs/tidb_http_api.md [hunk 2]
```diff
@@ -159,4 +159,13 @@ timezone.*
     curl -X POST http://{TiDBIP}:10080/ddl/owner/resign
     ```
     
+1. Get TiDB all ddl job history information.
```
Comment (winkyao) [inline docs/tidb_http_api.md]: Get all TiDB DDL job history information.

Comment (ciscoxll) [reply docs/tidb_http_api.md]: @winkyao Done.

**Pull request merged**